VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -39,6 +39,7 @@
             <LinearLayout
                 android:id="@+id/last_access_bar"
                 android:orientation="vertical"
+                android:layout_height="wrap_content"
```

Comment:
`layout_height` is already defined in `style/last_access_bar`. If we're going to change it, I think we should change it there.